misk/src/main/kotlin/misk/web/exceptions/GrpcExceptionMapper.kt

```diff
@@ -0,0 +1,44 @@
+package misk.web.exceptions
+
+import com.squareup.wire.GrpcException
+import com.squareup.wire.GrpcStatus
+import misk.proto.Status
+import misk.web.Response
+import misk.web.ResponseBody
+import misk.web.mediatype.MediaTypes
+import misk.web.toResponseBody
+import okhttp3.Headers
+import org.slf4j.event.Level
+import wisp.logging.getLogger
+import java.io.IOException
+import java.net.HttpURLConnection
+import javax.inject.Inject
+
+internal class GrpcExceptionMapper @Inject internal constructor() : ExceptionMapper<GrpcException> {
+  private val logger = getLogger<GrpcExceptionMapper>()
+
+  override fun toResponse(th: GrpcException) = INTERNAL_SERVER_ERROR_RESPONSE
```

Comment:
Would it be possible to have this class inherit everything from `IOExceptionMapper` and only override the `loggingLevel`?

Reply:
I suspect the answer is no, since ExceptionMapper has a type parameter that is different.

Reply:
I think you're right—if I'm not mistaken it would be possible in Java.